diktat-rules/src/main/kotlin/org/cqfn/diktat/ruleset/rules/files/NewlinesRule.kt

```diff
@@ -147,7 +139,23 @@ class NewlinesRule(private val configRules: List<RulesConfig>) : Rule("newlines"
 
         // We need to check newline only if prevCodeSibling exists. It can be not the case for unary operators, which are placed
         // at the beginning of the line.
-        if (node.prevCodeSibling()?.isFollowedByNewline() == true) {
+        if (node.firstChildNode != null && node.firstChildNode?.elementType == ELVIS) {
```

Comment:
```suggestion
        if (node.firstChildNode?.elementType == ELVIS) {
```
This check is redundant